Sources/Apodini/Apodini.docc/GettingStarted/DatabaseConnection.md

```diff
@@ -0,0 +1,190 @@
+# Database Connection
```

Comment:
Maybe it makes sense to move this to additional functionality and refer to the articles there from the getting started section?